src/admin/participants/handler.js

```diff
@@ -161,6 +162,11 @@ const update = async function (request) {
   Sidecar.logRequest(request)
   try {
     const updatedEntity = await Participant.update(request.params.name, request.payload)
+    if (request.payload.isActive !== undefined) {
+      const isActiveText = request.payload.isActive ? 'activated' : 'disabled'
```

Comment:
Should this be hardcoded or put into the enum?

Reply:
I see your point and I've created local enum object. I don't think we should flood the global Enum with debugging messages, but probably have a story in future where we isolate all debug messages into a single file, which would later ease i18n. @elnyry, please backlog if appropriate.